packages/db/src/schema/page.ts

```diff
@@ -0,0 +1,27 @@
+import { mysqlTable, varchar, int, timestamp } from "drizzle-orm/mysql-core";
+import { relations } from "drizzle-orm";
+import { incident } from "./incident";
+import { statusJob } from "./status-job";
+
+export const page = mysqlTable("page", {
+  id: int("id").autoincrement().primaryKey(),
+
+  workspaceId: int("workspace_id").notNull(),
+
+  slug: varchar("slug", { length: 256 }), // which is used for https://slug.openstatus.dev
```

Comment:
We should store the `slug` inside a `workspace`. So that every company has only a single status page with all the websites.

Reply:
I thought we could have multiple page for a workspace :)